src/oas2/transformers/__tests__/servers.test.ts

```diff
@@ -94,4 +99,21 @@ describe('translateToServers', () => {
       },
     ]);
   });
+
+  test('should handle invalid server host gracefully', () => {
+    expect(translateToServers({ host: 123 as any, basePath: '/base-path' }, { schemes: ['http', 'https'] })).toEqual(
```

Comment:
Why not handle it as a string?

```ts
[
  {
    url: 'http://123'
  }
]
```

Reply:
I think OK to just toss it - that would be a very odd server host ha.

Reply:
Users do very _odd_ things with their OpenAPI and not much extra effort to support it lol 🤷‍♂

Reply:
From a Prism perspective, we do not use servers anyway so whatever works for you will work for us as well.